lib/MusicBrainz/Server/Form/Field/ArtistCredit.pm

```diff
@@ -123,7 +123,14 @@ sub json {
     my $artists = $c->model('Artist')->get_by_ids(map { $_->{artist}->{id} } @$names);
     for my $name (@$names) {
         $name->{artist}->{entityType} = 'artist';
-        $name->{artist}->{gid} = $artists->{$name->{artist}->{id}}->gid if $artists->{$name->{artist}->{id}};
+        my $artist = $artists->{$name->{artist}->{id}};
+        if ($artist)
+        {
```

Comment:
Whooops...at least the heading on that page is accurate: "Status: This page is work in progress and has very questionable content." :)

I'll fix it to be less questionable.